.istanbul.yml

```diff
@@ -0,0 +1,2 @@
+instrumentation:
+  extensions: ['.js', '.jsx']
```

Comment:
you can move your coverage values into this file too
```
check:
  global:
    statements: 100
    lines: 100
    branches: 100
    functions: 100
```
keep the package.json a bit neater